types/react-helmet/index.d.ts

```diff
@@ -48,6 +48,11 @@ export interface HelmetDatum {
     toComponent(): React.Component<any, any>;
 }
 
+export interface HelmetAttributesTagObject {
+    toString(): string;
+    toComponent(): React.HTMLAttributes<HTMLBodyElement | HTMLElement>;
+}
+
```

Comment:
Would it be worth going through the trouble to have another, more explicit interface for bodyAttrs as well as htmlAttrs? Maybe all extending a common interface ?